packages/strapi-design-system/src/Card/CardAsset.js

```diff
@@ -26,6 +26,8 @@ const CardAssetWrapper = styled.div`
   width: 100%;
   background: repeating-conic-gradient(${({ theme }) => theme.colors.neutral100} 0% 25%, transparent 0% 50%) 50% / 20px
     20px;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
```

Comment:
What do you think about using `theme.borderRadius` instead, to keep this value consistent with all others?

Reply:
nice catch - sorry I keep forgetting about this one